### PR TITLE
workflows: custom search factory

### PR DESCRIPTION
**Pull request by jalavik**
This is the custom searcher for the new angular based Holding Pen list view filtering out some of the JSON for performance reasons.

### CODE REVIEW

File: inspirehep/modules/workflows/search.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+from __future__ import absolute_import, print_function, division
+
+from invenio_workflows_ui.search import default_search_factory
+
+
+def holdingpen_search_factory(self, search, **kwargs):
+    """Override search factory."""
+    search, urlkwargs = default_search_factory(self, search, **kwargs)
+    search = search.extra(_source={"include": ['_workflow', 'metadata']})
```
Comment (jalavik) [inline inspirehep/modules/workflows/search.py]: @eamonnmag can you provide a list of includes we want for the list view?

Comment (eamonnmag) [inline inspirehep/modules/workflows/search.py]: I currently have 

``` python
search = search.extra(_source={
        'include': ['titles', 'abstracts', '_workflow', 'subject_terms',
                    'authors', 'name', '_extra_data.relevance_prediction']})
```

Comment (jalavik) [inline inspirehep/modules/workflows/search.py]: Thanks, updated!

File: tests/integration/conftest.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def teardown():
         migrate('./inspirehep/demosite/data/demo-records.xml.gz', wait_for_results=True)
         es.indices.refresh('records-hep')  # Makes sure that all HEP records were migrated.
 
-        add_citation_counts(request_timeout=40)
+        add_citation_counts(request_timeout=80)
```
Comment (jalavik) [inline tests/integration/conftest.py]: @jacquerie Trying here what you mentioned IRL as it is blocking atm.

Comment (jacquerie) [inline tests/integration/conftest.py]: Ah, I didn't see this comment. 👍

**Pull request merged**